lib/swaps/SwapClientManager.ts

```diff
@@ -0,0 +1,224 @@
+import Config from '../Config';
+import { Models } from '../db/DB';
+import BaseClient from '../BaseClient';
+import LndClient from '../lndclient/LndClient';
+import { LndLogger, LndPubKeys, LndInfos } from '../lndclient/types';
+import RaidenClient from '../raidenclient/RaidenClient';
+import { RaidenInfo } from '../raidenclient/types';
+import Logger, { Loggers } from '../Logger';
+import Pool from '../p2p/Pool';
+import { SwapClient } from '../constants/enums';
+import OrderBookRepository from '../orderbook/OrderBookRepository';
+import errors from './errors';
+import { Currency } from '../orderbook/types';
+
+class SwapClientManager {
+  /** A map between currencies and all swap clients */
+  private swapClients = new Map<string, BaseClient>();
+  private raidenClient!: RaidenClient;
```

Comment:
I would either make this nullable with the `?` operator, or create the `RaidenClient` object in the constructor (so that indeed this value will never be undefined) and then initialize it inside the `init` function here and let it become disabled.